bungee/src/main/java/me/leoko/advancedban/bungee/listener/CommandReceiverBungee.java

```diff
@@ -11,8 +11,12 @@
 
 public class CommandReceiverBungee extends Command {
 
-    public CommandReceiverBungee(String name) {
-        super(name);
+    /**
```

Comment:
I would have liked to use @Nullable but because it's available transitively, but it's not used anywhere